app/src/main/java/com/swent/assos/ui/screens/manageAsso/ManageAssociation.kt

```diff
@@ -0,0 +1,158 @@
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.snapshotFlow
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.swent.assos.model.data.Association
+import com.swent.assos.model.navigation.Destinations
+import com.swent.assos.model.navigation.NavigationActions
+import com.swent.assos.model.view.AssoViewModel
+import com.swent.assos.ui.components.EventItem
+import com.swent.assos.ui.components.NewsItem
+
+@Composable
+fun ManageAssociation(assoId: String, navigationActions: NavigationActions) {
+  val viewModel: AssoViewModel = hiltViewModel()
+
+  val association by viewModel.association.collectAsState()
+  val news by viewModel.news.collectAsState()
+  val events by viewModel.events.collectAsState()
+
+  val listStateNews = rememberLazyListState()
+  val listStateEvents = rememberLazyListState()
+
+  LaunchedEffect(key1 = Unit) {
+    viewModel.getAssociation(assoId)
+    viewModel.getNews(assoId)
+    viewModel.getEvents(assoId)
+  }
+
+  LaunchedEffect(listStateNews) {
+    snapshotFlow { listStateNews.layoutInfo.visibleItemsInfo }
+        .collect { visibleItems ->
+          if (visibleItems.isNotEmpty() && visibleItems.last().index == news.size - 1) {
+            viewModel.getMoreNews(assoId)
+          }
+        }
+  }
+
+  LaunchedEffect(listStateEvents) {
+    snapshotFlow { listStateEvents.layoutInfo.visibleItemsInfo }
+        .collect { visibleItems ->
+          if (visibleItems.isNotEmpty() && visibleItems.last().index == events.size - 1) {
+            viewModel.getMoreEvents(assoId)
+          }
+        }
+  }
+
+  LazyColumn(
```

Comment:
I do not think you need a `LazyColumn` either here
```suggestion
  Column(
```